Error: Cannot find module 'express-promise-router'  in step 04-add-aad-auth.md of tutorial

### Where did you get the code?

- [ ] Downloaded from GitHub
- [ ] Downloaded from the [Microsoft Graph quick start tool](https://developer.microsoft.com/graph/quick-start)
- [X] Followed the tutorial from [Microsoft Graph tutorials](https://docs.microsoft.com/graph/tutorials)

### Describe the bug

When starting the web server, just before the section **Get user details**, the nom start fails with _Error: Cannot find module 'express-promise-router'_

### To Reproduce

Steps to reproduce the behavior:

1. Follow the tutorial until step 4
1. Start the webserver with ´npm start`

### Expected behavior

The webserver starts

### Additional context

running `npm install express-promise-router` fixes the issue 

Btw: you have copied the `<MsalInitSnippet>` in the demo AFTER `var app = express();`, whereas the tutorial says before `var indexRouter = require('./routes/index');` - which gives another error. Demo file here in GitHub seems fine (and more logical)

--- a/tutorial/02-create-app.md
+++ b/tutorial/02-create-app.md
@@ -55,7 +55,7 @@ Before moving on, install some additional packages that you will use later:
 1. Run the following command in your CLI.
 
     ```Shell
-    npm install dotenv@8.2.0 moment@2.29.1 moment-timezone@0.5.31 connect-flash@0.1.1 express-session@1.17.1 isomorphic-fetch@3.0.0
+    npm install dotenv@8.2.0 moment@2.29.1 moment-timezone@0.5.31 connect-flash@0.1.1 express-session@1.17.1 express-promise-router@4.0.1 isomorphic-fetch@3.0.0
     npm install @azure/msal-node@1.0.0-beta.0 @microsoft/microsoft-graph-client@2.1.1 windows-iana@4.2.1 express-validator@6.6.1 uuid@8.3.1
     ```
 

--- a/tutorial/04-add-aad-auth.md
+++ b/tutorial/04-add-aad-auth.md
@@ -19,7 +19,7 @@ In this exercise you will extend the application from the previous exercise to s
 
 ## Implement sign-in
 
-1. Locate the line `var indexRouter = require('./routes/index');` in **./app.js**. Insert the following code **before** that line.
+1. Locate the line `var app = express();` in **./app.js**. Insert the following code **after** that line.
 
     :::code language="javascript" source="../demo/graph-tutorial/app.js" id="MsalInitSnippet":::
 

